### PR TITLE
Only autofocus inputs when document width is above threshold

### LINKED ISSUE #170: Prevent password input autofocus on mobile

### PR DESCRIPTION
**Pull request by nibhar**
This PR checks if the document is wieder than 600px before automatically focusing an input field.

Resolves #170

### CODE REVIEW

File: src/components/PassphraseInput.js [hunk 1]
```diff
@@ -27,7 +27,10 @@ class PassphraseInput extends Nimiq.Observable {
 
         this.$input.placeholder = placeholder;
 
-        this.$eyeButton.addEventListener('click', () => this._changeVisibility());
+        this.$eyeButton.addEventListener('click', () => {
+            this._changeVisibility();
+            this.focus();
```
Comment (sisou) [inline src/components/PassphraseInput.js]: Can we find out, if the input was focused before visibility was changed, to only re-focus when it had focus before? Or do you think it's expected to open the keyboard when I just click the visibility button on mobile?

File: src/request/export/ExportFile.js [hunk 2]
```diff
@@ -48,12 +49,23 @@ class ExportFile extends Nimiq.Observable {
             alert('Wallet Files are not yet implemented.');
             this._finish();
         });
+        window.addEventListener('hashchange', event => {
+            const newUrl = new URL(event.newURL);
+            if (newUrl.hash === `#${ExportFile.Pages.EXPORT_FILE}`) {
+                this._downloadKeyFilePassphraseBox.reset();
+                if (TopLevelApi.getDocumentWidth() > Constants.MIN_WIDTH_FOR_AUTOFOCUS) {
+                    this._downloadKeyFilePassphraseBox.focus();
```
Comment (sisou) [inline src/request/export/ExportFile.js]: Why is this necessary?

File: src/components/PassphraseInput.js (hunk 1 above)
Comment (nibhar) [reply src/components/PassphraseInput.js]: Functionality of the Eye Button is unchanged. I agree that we should talk about it.

File: src/request/export/ExportFile.js (hunk 2 above)
Comment (nibhar) [reply src/request/export/ExportFile.js]: ExportFiles Password Box would not reset in case a user navigates to ExportWords and then back without it.
